readme.txt

```diff
@@ -105,105 +105,87 @@ There are opportunities for developers at all levels to contribute. [Learn more
 
 = 4.7 =
 
-* Release date:
-* Release post:
+**Performance Improvements**
 
-**New Features and Improvements**
+* Experience better performance with single page load caching of the media summary.
+* We made some improvements to avoid slow queries on sites with thousands of images.
+* The Top Posts widget now utilizes an endpoint from the WP.com REST API improving performance, and update flexibility.
+* Improve development mode performance by only calling `site_url()` once.
+* We rewrote the way major/minor rules lists are generated to save bandwidth, and memory for sites with lots of widgets.
+* We've removed sync options that weren't need to save memory during the sync process.
 
-**Performance Improvements**
+**Accessibility**
 
-* Development mode: improve performance by only calling `site_url()` once. #6130
-* Media Summary: improve performance with single page load caching. #5938
-* Open Graph: Avoid slow queries on sites with thousands of images. #6297
-* Widgets: update the Top Posts Widget to retrieve posts via the WordPress.com REST API. #6335
-* Widget Visibility: Rewrite the way that the major/minor rules lists are generated to save bandwidth and memory. #6333
+* We've improved the highlight of the stats range for a better visual experience and to make it consistent with other areas of the dashboard.
+* Added a missing label to one of the fields in the Email sharing dialog.
+* We've enabled keyboard focus styling in the new admin interface.
+* Increased padding to sharing buttons on mobile to avoid usability issues.
+* We've replaced text labels with icons to improve usability on smaller devices.

```

Comment:
It think we should mention which module this is about (Widget Visibility), as it's not super obvious imo.